curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-string-and-array-methods-by-building-a-music-player/655b4dad1d38ff7cdd65cbfe.md

```diff
@@ -680,6 +680,10 @@ const shuffle = () => {
   userData.currentSong = null;
   userData.songCurrentTime = 0;
 
+  if (userData?.songs.length === 0) {
+    return;
```

Comment:
Can we have the optional chaining for songs property as well ?
`userData?.songs?.length === 0`